### PR TITLE
Added second confirmation during deleting product or service

### PR DESCRIPTION
**Pull request by konadrian**
DeleteInv

### CODE REVIEW

File: modules/Vtiger/models/InventoryDelete.php [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+	private $moduleNameForDelete = '';
+
+	/** @var array Records where record for delete is set */
+	private $recordsWhereInvIsSet = [];
+
+	/** @var array Number of related records for display user */
+	private $recordsLimit = 10;
+
+	/** @var int Record for delete id */
+	private $recordId = 0;
```
Comment (mariuszkrzaczkowski) [inline modules/Vtiger/models/InventoryDelete.php]:
```suggestion
	private $recordId;
```

File: modules/Vtiger/models/InventoryDelete.php [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+ * @author    Adrian Kon <a.kon@yetiforce.com>
+ */
+
+/**
+ * Check is inventory exists in related records class.
+ */
+class Vtiger_InventoryDelete_Model extends \App\Base
+{
+	/** @var string Module name for delete */
+	private $moduleNameForDelete = '';
```
Comment (mariuszkrzaczkowski) [inline modules/Vtiger/models/InventoryDelete.php]:
```suggestion
	private $moduleNameForDelete;
```

**Review by mariuszkrzaczkowski — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
## Black Duck Security Report

Merging #16272 into developer will increase security risk!

### Added Components

Low Risk:    1
Clean:       67

### Removed Components

Clean:       52

[Click here to see full report](https://copilot.blackducksoftware.com/github/repos/YetiForceCompany/YetiForceCRM/pull-requests/16272)

File: modules/Vtiger/models/InventoryDelete.php [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+	/**
+	 * Get records where deleted record is set.
+	 *
+	 * @return array
+	 */
+	public function getRecordsWhereRecordIsSet(): array
+	{
+		$this->recordsWhereInvIsHtml .= '<uL>';
+		$allModules = \vtlib\Functions::getAllModules(false, true);
```
Comment (mariuszkrzaczkowski) [inline modules/Vtiger/models/InventoryDelete.php:59]:
```suggestion
		$allModules = \vtlib\Functions::getAllModules(true, true);
```

**Pull request closed without merging**